packages/checkbox/src/Checkbox.ts

```diff
@@ -28,72 +30,98 @@ import '@spectrum-web-components/icons-ui/icons/sp-icon-dash75.js';
-        <sp-icon-checkmark300
-            id="checkmark"
-            class="spectrum-Icon spectrum-UIIcon-Checkmark300"
-        ></sp-icon-checkmark300>
-    `,
+    s: () => {
+        return html`
+            <sp-icon-checkmark75
+                id="checkmark"
+                class="spectrum-Icon spectrum-UIIcon-Checkmark75"
+            ></sp-icon-checkmark75>
+        `;
+    },
+    m: () => {
+        return html`
+            <sp-icon-checkmark100
+                id="checkmark"
+                class="spectrum-Icon spectrum-UIIcon-Checkmark100"
+            ></sp-icon-checkmark100>
+        `;
+    },
+    l: () => {
+        return html`
+            <sp-icon-checkmark200
+                id="checkmark"
+                class="spectrum-Icon spectrum-UIIcon-Checkmark200"
+            ></sp-icon-checkmark200>
+        `;
+    },
+    xl: () => {
+        return html`
+            <sp-icon-checkmark300
+                id="checkmark"
+                class="spectrum-Icon spectrum-UIIcon-Checkmark300"
+            ></sp-icon-checkmark300>
+        `;
+    },
 };
 
 const dashIcon = {
-    s: html`
-        <sp-icon-dash75
-            id="partialCheckmark"
-            class="spectrum-Icon spectrum-UIIcon-Dash75"
-        ></sp-icon-dash75>
-    `,
-    m: html`
-        <sp-icon-dash100
-            id="partialCheckmark"
-            class="spectrum-Icon spectrum-UIIcon-Dash100"
-        ></sp-icon-dash100>
-    `,
-    l: html`
-        <sp-icon-dash200
-            id="partialCheckmark"
-            class="spectrum-Icon spectrum-UIIcon-Dash200"
-        ></sp-icon-dash200>
-    `,
-    xl: html`
-        <sp-icon-dash300
-            id="partialCheckmark"
-            class="spectrum-Icon spectrum-UIIcon-Dash300"
-        ></sp-icon-dash300>
-    `,
+    s: () => {
+        return html`
+            <sp-icon-dash75
+                id="partialCheckmark"
+                class="spectrum-Icon spectrum-UIIcon-Dash75"
+            ></sp-icon-dash75>
+        `;
+    },
+    m: () => {
+        return html`
+            <sp-icon-dash100
+                id="partialCheckmark"
+                class="spectrum-Icon spectrum-UIIcon-Dash100"
+            ></sp-icon-dash100>
+        `;
+    },
+    l: () => {
+        return html`
+            <sp-icon-dash200
+                id="partialCheckmark"
+                class="spectrum-Icon spectrum-UIIcon-Dash200"
+            ></sp-icon-dash200>
+        `;
+    },
+    xl: () => {
+        return html`
+            <sp-icon-dash300
+                id="partialCheckmark"
+                class="spectrum-Icon spectrum-UIIcon-Dash300"
+            ></sp-icon-dash300>
+        `;
+    },
 };
 
 /**
  * @element sp-checkbox
  * @slot - content to display as the label for the Checkbox
  * @fires change - Announces a change in the `checked` property of a Checkbox
  */
-export class Checkbox extends SizedMixin(CheckboxBase, {
+export class Checkbox extends SizedMixin(CheckboxMixin(SpectrumElement), {
```

Comment:
@lazd this removes `focusable`, but only for Checkbox.

Reply:
Huh, but `CheckboxMixin` still extends `Focusable`, right? So it's not removed?